configure

```diff
@@ -554,6 +554,18 @@ parser.add_option('-C',
     dest='compile_commands_json',
     help=optparse.SUPPRESS_HELP)
 
+# Electron-specific. Used to assist in calling the gyp build from GN.
+parser.add_option('--config-out-dir',
+    action='store',
+    default='.',
+    dest='config_out_dir',
+    help='output directory to write .gypi files to')
+
+parser.add_option('--no-run-gyp',
```

Comment:
(minor) might add the `# Electron-specific` comment before this option as well?

When reading the code raw, it's not obvious that the comment on 557 applies to both `--config-out-dir` and `--no-run-gyp`